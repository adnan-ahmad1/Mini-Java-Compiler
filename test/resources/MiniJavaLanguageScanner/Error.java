int a = 4;
int b = 6;

boolean t = a > b;

System.out.println(t);