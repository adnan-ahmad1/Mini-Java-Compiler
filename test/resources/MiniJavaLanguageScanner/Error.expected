INT ID(a) BECOMES INTEGER(4) SEMICOLON INT ID(b) BECOMES INTEGER(6) SEMICOLON BOOLEAN ID(t) BECOMES ID(a) <UNEXPECTED(>)> ID(b) SEMICOLON PRINTLN LPAREN ID(t) RPAREN SEMICOLON